Sources/Sentry/SentryTracer.m

```diff
@@ -151,8 +151,7 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
     if (_configuration.profilesSamplerDecision.decision == kSentrySampleDecisionYes) {
         _isProfiling = YES;
         _startSystemTime = SentryCurrentDate.systemTime;
-        [SentryProfiler startWithHub:hub];
-        trackTracerWithID(self.traceId);
```

Comment:
We now just call the bookkeeping function `trackTracerWithID` from `SentryProfiler`'s implementation (the function was renamed to `trackProfilerForTracer(SentryProfiler *profiler, SentryTracer *tracer)`